profiles/package.mask

```diff
@@ -29,6 +29,10 @@
 
 #--- END OF EXAMPLES ---
 
+# Anestis Zaganidis <anestis.zag@gmail.com> (4 Dec 2018)
+# llpp builds from git
+=app-text/llpp-30
```

Comment:
Why masking the package?

Reply:
It was recommended by repoman because it pulls the sources from git. However, it points to a specific commit. Should I remove it from mask?

Reply:
I changed the ebuild to get a tar snapshot. I removed the mask.